aws-lc-sys/builder/main.rs

```diff
@@ -386,6 +388,10 @@ fn is_no_asm() -> bool {
     unsafe { AWS_LC_SYS_NO_ASM }
 }
 
+fn allow_prebuilt_nasm() -> Option<bool> {
+    unsafe { AWS_LC_SYS_PREBUILT_NASM }
+}
```

Comment:
Any reason to not just do:

```suggestion
fn allow_prebuilt_nasm() -> bool {
    unsafe { AWS_LC_SYS_PREBUILT_NASM } == Some(true)
}
```

Reply:
The return value here needs to represent 3 possibilities: (1) the environment variable is set to *true*, (2) the environment variable is set to *false*, and (3) the environment variable is not set.